css/airspace.css

```diff
@@ -227,7 +227,18 @@ font header .navbar-default .navbar-nav li a:hover {
   padding: 90px 0;
 }
 #service .service-item {
-  padding-bottom: 30px;
+  display: flex;
```

Comment:
We're trying to put new CSS into custom.css as to not confuse future maintainers.

Please solve this in a way that doesn't change the mainstream airspace.css file.